main.tf

```
@@ -35,6 +35,16 @@ resource "aws_s3_bucket" "private_bucket" {
     }
   }
 
+  lifecycle_rule {
+    enabled = true
+
+    prefix = "_AWSBucketInventory/"
+
+    expiration {
+      days = 7
```

Sorry, I missed this in the first review. I'm wondering if we should add some buffer to this if we default to running reports weekly. perhaps bump this to 14 days?
```suggestion
      days = 14
```